app/controllers/lunches/details/index.js

```diff
@@ -148,7 +148,7 @@ function showCamera() {
 	}
 	
 	// Send red sqare on Simulator
-	if (Ti.App.getDeployType() == 'development') {
+	if (utils.isEmulator()) {
```

Comment:
Really? That should *not* happen. Ti.App.getDeployType() `development` is only supposed to return `true` for sim. For devices, it would do `test` and for production `production.`